src/dataset-operator/cmd/manager/main.go

```diff
@@ -126,6 +128,14 @@ func main() {
 		log.Error(err, "")
 		os.Exit(1)
 	}
+	if err := cephv1.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "")
+		os.Exit(1)
+	}
+	if err := objectstorage.AddToScheme(mgr.GetScheme()); err != nil {
+		log.Error(err, "")
+		os.Exit(1)
+	}
```

Comment:
Why is rook ceph referenced in the main? This should be part of the caching plugin, please check this is fixed throughout this PR.

Reply:
Good catch, the logic/imports for the ceph plugin should not be on the main dataset operator

Reply:
Check the last commit https://github.com/IBM/dataset-lifecycle-framework/pull/39/commits/a20c83c7934ed1e6f5fed082521e2edef333a21a

Reply:
Perfect, thanks!